operators/src/source/gdal_source.rs

```diff
@@ -163,6 +163,56 @@ pub struct GdalDatasetParameters {
     pub gdal_config_options: Option<Vec<(String, String)>>,
 }
 
+/// A user friendly representation of Gdal's geo transform. In contrast to [`GeoTransform`] this
+/// geo transform allows arbitrary pixel sizes and can thus also represent rasters where the origin is not located
+/// in the upper left corner. It should only be used for loading rasters with Gdal and not internally.
+#[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GdalDatasetGeoTransform {
+    pub origin_coordinate: Coordinate2D,
+    pub x_pixel_size: f64,
+    pub y_pixel_size: f64,
+}
+
+/// Default implementation for testing purposes where geo transform doesn't matter
+impl Default for GdalDatasetGeoTransform {
+    fn default() -> Self {
+        Self {
+            origin_coordinate: (0.0, 0.0).into(),
+            x_pixel_size: 1.0,
+            y_pixel_size: -1.0,
+        }
+    }
+}
```

Comment:
I guess we shouldn't implement Default for testing purposes. Maybe we could create a trait TestingDefault. On the other hand, maybe we should use more different values for catching bugs :shrug:.

Reply:
7a6766483729089fac8b6f0ffcb2c6b1daf168c1